pkg/apis/componentconfig/v1alpha1/defaults.go

```diff
@@ -112,3 +132,193 @@ func SetDefaults_LeaderElectionConfiguration(obj *LeaderElectionConfiguration) {
 		obj.RetryPeriod = unversioned.Duration{Duration: 2 * time.Second}
 	}
 }
+
+func SetDefaults_KubeletConfiguration(obj *KubeletConfiguration) {
```

Comment:
I think this should happen automatically. We are just binding the flags to different addresses (fields in the componentconfig struct). The man page generation flow should be unaffected.